CONTRIBUTING.md

```diff
@@ -74,7 +74,7 @@ For more information about virtual environment see [here](https://docs.python.or
 3. Activate the venv
     
     ```
-    source .venv/bin/activate
+    source venv/bin/activate
```

Comment:
It's wrong, or to say it better a bit out of python convention. I've just checked and you are right in Makefile it's venv, but as said above it should be .venv. I'm rejecting this PR, the one which need fixes is Makefile.
@ehsavoie feel free to fix the Makefile :) BTW you can also add the target to activate the venv while you are there ;)